Mono.Cecil/ModuleDefinition.cs

```diff
@@ -333,7 +336,12 @@ public sealed class ModuleDefinition : ModuleReference, ICustomAttributeProvider
 			set { characteristics = value; }
 		}
 
-		[Obsolete("Use FileName")]
+		public ushort Linker {
```

Comment:
I don't think we need to expose this for now, I suggest we handle this like the timestamp.